### ISSUE TITLE
Yr.no shows night weather during the day

### ISSUE THREAD

**Issue by Madis0** [tommasoberlose/another-widget#326]
Yr.no seems to show the night weather condition (moon icon) and temperature during the day. The temperature does not match the current one (as seen by the Yr app), so I think it is offset by some hours. 

Tried refreshing the widget and increasing the weather update interval, did not help.

**Comment by Moutony** [comment]
Do you live in a time zone with a 9-hour difference compared with Norway (Yr.no is Norwegian) ?

**Comment by Madis0** [comment]
@Moutony No, the difference is only one hour. Either way the app should be able to convert the time accordingly.

**Comment by agraaaaao** [comment]
I'm having this issue as well

**Comment by karaca77** [comment]
same bug, the highest temperature value is displayed instead of the hourly temperature and same icon stays all day
